### DISCUSSION TITLE
Nice job

### DISCUSSION THREAD

**Discussion by Kartoffelbreili** [complexlogic/EasyAudioSync#1] [General]
Tried out your program. Works flawlessly. Nice job. This will be able to replace Musicbee for transcoding to my secondary library that i sync with resilio. Directly to phone would be neat but i heard that this is kind of slow usually when you dont program a mobile app. Got that from Steven Mayall. 
One more codec that would be nice to have is wma. They are rare in my library but they just need to be copied.

**Comment by complexlogic** [comment]
Glad to hear it worked for you.

The program actually does have copy-only WMA support. I didn't add it to the settings because there's nothing to change. Did you check to see if your WMA files were copied correctly?

**Reply by Kartoffelbreili** [reply]
Well i didnt actually check i just saw it wasnt on the gui. But it does work. Thanks
